deepinterpolation/generator_collection.py

```diff
@@ -897,112 +918,134 @@ class OphysGenerator(SequentialGenerator):
     def __init__(self, json_path):
         "Initialization"
         super().__init__(json_path)
+        self._gpu_available = tf.test.is_gpu_available()
+        self._movie_data = None
+        if self._gpu_available:
+            self._cached_index = None
```

Comment:
Please rename to something clearer like `_prev_batch_index`

Reply:
It's the current batch index

Reply:
So is `_current_batch_index` a clearer name?

Reply:
I set it to `_batch_tensor_index`. I wanted to specify that it specifically refers to the index for `self._batch_tensor`. I think current is implied by the name? I added a comment next to it to clarify for anyone reading the code.